cost/aws/native_utilization/native_utilization_ap_south_1_no_cloud_in_rs

```diff
@@ -0,0 +1,421 @@
+name "AWS Native Utilization"
+rs_pt_ver 20180301
+type "policy"
+short_description "Checks for buckets that are open to everyone. See the [README](https://github.com/rightscale/policy_templates/tree/master/security/storage/aws/public_buckets) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
+long_description "Version: 1.0"
+severity "low"
+category "Cost"
+
+###############################################################################
+# Permissions
+###############################################################################
+
+permission "perm_read_creds" do
+  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
```

Comment:
Last week we added a new pattern for parameters.  All parameters should have a label and description.  The email address param needs to be changed and you probably need to add the exclude tag

```
parameter "param_email" do

  type "list"

  label "Email addresses to notify"

  description "Email addresses of the recipients you wish to notify when new incidents are created"

end


 


parameter "param_exclude_tags" do

  type "list"

  label "Tags to ignore"

  description "List of tags that will exclude resources from being evaluated by this policy. Multiple tags are evaluated as an 'OR' condition. Tag keys or key/value pairs can be listed. Example: 'test,env=dev'"

end
```